frigate/object_processing.py

```diff
@@ -515,6 +516,39 @@ def get_current_frame(self, draw_options={}):
                     thickness = 5
                     color = self.config.model.colormap[obj["label"]]
 
+                    # debug autotracking zooming - show the zoom factor box
+                    if (
+                        self.camera_config.onvif.autotracking.zooming
```

Comment:
Always show this or only when auto tracking an object?

Reply:
Only when actively autotracking an object and zooming is enabled. https://github.com/blakeblackshear/frigate/blob/5b7ff9a8498469f0831ad39c0bd633f81299f363/frigate/object_processing.py#L501